src/common/components/clusterDetail/ClusterProgress.tsx

```diff
@@ -68,154 +47,20 @@ const getInstallationStatus = (
-
-  return (
-    <Flex className="pf-u-mr-3xl">
-      <FlexItem>{icon}</FlexItem>
-      <FlexItem>{text}</FlexItem>
-    </Flex>
-  );
-};
-
-const getFinalizingStatusIcon = (cluster: Cluster) => {
-  let statusIcon;
-  const { monitoredOperators = [] } = cluster;
-  const areAllBuiltInOperatorsAvailable = monitoredOperators.length
-    ? monitoredOperators
-        .filter((op) => op.operatorType === 'builtin')
-        .every((op) => op.status === 'available')
-    : false;
-  if (areAllBuiltInOperatorsAvailable) {
-    statusIcon = <CheckCircleIcon color={okColor.value} />;
-  } else {
-    switch (cluster.status) {
-      case 'finalizing':
-        statusIcon = <InProgressIcon />;
-        break;
-      case 'error':
-      case 'cancelled':
-        statusIcon = <ExclamationCircleIcon color={dangerColor.value} />;
-        break;
-      case 'installed':
-      case 'adding-hosts':
-        statusIcon = <CheckCircleIcon color={okColor.value} />;
-        break;
-      default:
-        statusIcon = <PendingIcon />;
-    }
-  }
-
-  return statusIcon;
-};
-
-type FinalizingProgressProps = {
-  cluster: Cluster;
-  onFetchEvents: EventListFetchProps['onFetchEvents'];
-  fallbackEventsURL?: string;
-};
-
-const FinalizingProgress: React.FC<FinalizingProgressProps> = ({
-  cluster,
-  onFetchEvents,
-  fallbackEventsURL,
-}) => {
-  const { status } = cluster;
-  const [isModalOpen, setIsModalOpen] = React.useState(false);
-  const closeModal = () => setIsModalOpen(false);
-  return (
-    <>
-      <EventsModal
-        title="Cluster Events"
-        isOpen={isModalOpen}
-        onClose={closeModal}
-        hostId={undefined}
-        cluster={cluster}
-        entityKind="cluster"
-        onFetchEvents={onFetchEvents}
-        fallbackEventsURL={fallbackEventsURL}
-      />
-      <Flex className="pf-u-mr-3xl">
-        <FlexItem>{getFinalizingStatusIcon(cluster)}</FlexItem>
-        <FlexItem>
-          {status === 'finalizing' ? (
-            <Popover
-              zIndex={300} // set the zIndex below Cluster Events Modal
-              headerContent={<>Initialization</>}
-              bodyContent={
-                <TextContent>
-                  <Text>
-                    This stage may take a while to finish. To view detailed information, click the
-                    events log link below.
-                  </Text>
-                </TextContent>
-              }
-              footerContent={
-                <Button variant={ButtonVariant.link} isInline onClick={() => setIsModalOpen(true)}>
-                  Open Events Log
-                </Button>
-              }
-            >
-              <Button variant={ButtonVariant.link} isInline>
-                Initialization
-              </Button>
-            </Popover>
-          ) : (
-            'Initialization'
-          )}
-        </FlexItem>
-      </Flex>
-    </>
-  );
-};
-
-type ClusterProgressProps = {
+export type ClusterProgressProps = {
```

Comment:
No need to export, we need to create ClusterProgressItemsProps anyway (see below)